src/kili/adapters/kili_api_gateway/helpers/queries.py

```diff
@@ -82,8 +85,13 @@ def execute_query_from_paginated_call(
                     )
                     payload = {"where": where, "skip": skip, "first": first}
                     elements = self._graphql_client.execute(query, payload)["data"]
+                    assert isinstance(
+                        elements, list
+                    ), "Paginated queries does not support operations that does not return a list"
+                    print(elements)
+                    elements: List = [format_json(elem, self.http_client) for elem in elements]
 
-                    if elements is None or len(elements) == 0:
+                    if len(elements) == 0:
```

Comment:
is it guaranteed that elements can't be None?

Reply:
yes by the isinstance(elements, list) just above